pubsub.go

```diff
@@ -1143,7 +1144,9 @@ func (p *PubSub) checkSigningPolicy(msg *Message) error {
 func (p *PubSub) publishMessage(msg *Message) {
 	p.tracer.DeliverMessage(msg)
```

Comment:
Should we trace such local messages or not?

Reply:
the tracer can see it's a local publication from the Local field -- but we might want to have some special casing in some tracers.
I think we should leave it as for now.